app/views/merge_requests/confirm_telephone_number.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :before_content do %>
   <% title = "Tell us if your organisation is merging" %>
   <% content_for :title, title %>
-  <%= govuk_back_link(href: :back) %>
+  <%= govuk_back_link href: absorbing_organisation_merge_request_path(id: @merge_request.requesting_organisation_id) %>
```

Comment:
Should this point to the merge_request id instead of the requesting_organisation_id `absorbing_organisation_merge_request_path(@merge_request)`? Do we have any tests for this?

Reply:
Hmm, I think this is routing to `/organisations/:id/merge-request` which is not related to a specific merge request, so should be fine. I see where it came from now!